api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/core/activation/Activations.kt

```diff
@@ -256,6 +270,7 @@ public enum class Activations {
                 HardSigmoid -> HardSigmoidActivation()
                 Swish -> SwishActivation()
                 Mish -> MishActivation()
+                HardShrink -> HardShrinkActivation(lower = -0.5f, upper = 0.5f)
```

Comment:
What do you think about moving these values as default values to the constructor?

Reply:
You are right. This will actually make much more sense. Thanks @zaleslaw